tmcprototype/DishLeafNode/DishLeafNode/DishLeafNode.py

```diff
@@ -243,7 +255,8 @@ def convert_radec_to_azel(self, data):
             #print("Hour angle in hours: ", katpoint._ephem_extra.angle_from_hours(hour_angle))
 
             # Geodetic latitude of the observer
-            latitude_degree_decimal = float(18) + float(31) / 60 + float(48) / (60 * 60)
+            #latitude_degree_decimal = float(18) + float(31) / 60 + float(48) / (60 * 60)
```

Comment:
Remove commented code or add TODO label

Reply:
Done. Th commit id is: [4d7c5fe8c492286f744029d161751676cffe0f1a](https://github.com/ska-telescope/tmc-prototype/commit/4d7c5fe8c492286f744029d161751676cffe0f1a)